app/controls/actionlabel.h

```diff
@@ -0,0 +1,24 @@
+#ifndef ACTIONLABEL_H
+#define ACTIONLABEL_H
+
+#include "controls/activelabel.h"
+
+class QAction;
+
+class ActionLabel : public ActiveLabel
```

Comment:
As far as I can see, the ActiveLabel class has no direct user anymore after your changes, right? If so, I would suggest to combine both classes into a single one.